### ISSUE TITLE
Please update the release history when publishing updates to VS Code

### ISSUE THREAD

**Issue by codykonior** [PowerShell/vscode-powershell#1878]
2.0.2 was just released but there is no information visible in VS Code about what has changed.

![image](https://user-images.githubusercontent.com/12396330/56117185-2319b380-5f9a-11e9-9623-43c922a9c429.png)

This is not very cool.

**Comment by rjmholt** [comment]
Thanks for opening an issue @codykonior.

The changelog you're seeing is for the most recent version. 2.0.0-preview.3 == 2.0.2. We have been using semver v2, which npm also uses, but the VSCode marketplace uses semver v1. We kept semver v2 to keep in step with other conventions in the PowerShell universe, but to be able to publish in the marketplace we've had to map the preview versions. This is only a preview-version problem, since the stable extension never has a prerelease label.

It's likely we'll change versioning scheme in the near future to eliminate confusion and to make our own lives easier as well, since managing the mismatch between npm versioning and VSCode marketplace versioning is complexity we want to do away with.
